documentation/changelogs/api/index.md

```diff
@@ -0,0 +1,10 @@
+# API changelogs
+
+Changelogs for <https://api.openverse.engineering/>.
+
+```{toctree}
+:glob:
+:titlesonly:
+
+*
+```
```

Comment:
This applies to all the index files in the changelog directories: the reason for this configuration is that it lists only the titles of the changelogs in ascending order. We might want to reverse this to have it list them in descending order so that the most recent changelog is at the top (as with most changelogs).

Reply:
I definitely find descending-order changelogs to be the most helpful, if that's a quick change to make!